frontend/src/components/volunteerLayout/index.js

```diff
@@ -115,6 +115,18 @@ function VolunteerLayout() {
             <img src={nav_item_settings} alt="Settings" />
             {!collapsed && "Settings"}
           </NavLink>
+
+
+          <NavLink
+            to="/volunteer/class-preferences"
```

Comment:
I'm assuming this is a temporary link to the page, however before we merge into main, let's put the temporary navigation somewhere on the volunteer profile page to match the design more closely and keep the navigation panel clean!
<img width="557" alt="Screenshot 2025-02-01 at 2 45 44 PM" src="https://github.com/user-attachments/assets/a2e9436a-bc39-4f75-8c60-b7eb3f2efd40" />